### ISSUE TITLE
Error in action=graph_edit: CMDPHP Validation Error, Variable:host_id

### ISSUE THREAD

**Issue by ABATAPA** [Cacti/cacti#2972]
When I open an edit of any graph (added by Data Query) and save (with or without change) I get an error.

**To Reproduce**
Console → Management → Graphs
Select any graph
Click "Unlock Graph"
Save

Got error:

(/graphs.php[45]:validate_graph_request_vars(),  /graphs.php[1818]:validate_store_request_vars(),  /lib/html_utility.php[682]:die_html_input_error(),  /lib/html_validate.php[64]:cacti_debug_backtrace())
Validation error for variable host_id with a value of c7304-amts. See backtrace below for more details.

**Expected behavior**
Correct editing at least of the title

**Screenshots**
![Screenshot_20190923_165138](https://user-images.githubusercontent.com/17951824/65431718-99acbe00-de22-11e9-9aba-8e1511df3661.png)
![Screenshot_20190923_165155](https://user-images.githubusercontent.com/17951824/65431727-9b768180-de22-11e9-828d-85c8c7aa8223.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Firefox
 - Version 69.0 (64 bit)

**Server**
Ubuntu 18.04.3 LTS
PHP 7.2.19-0ubuntu0.18.04.2 (cli) (built: Aug 12 2019 19:34:28) ( NTS )
Apache 2.4.29-1ubuntu4.11
Clean install 1.26 and upgrade to 1.3.0 (git)

**Additional context**
Reproduced in 1.2.6 and 1.3.0 (from git).

**Comment by cigamit** [comment]
This was resolved in the 1.2.x branch a while ago.  Please utilize it and then report back if you find different.

**Comment by cigamit** [comment]
Closing this for now.  If after updating to branch 1.2.x, you find that this problem is not resolved, just update the ticket and I can reopen.

**Comment by ABATAPA** [comment]
I wrote: «Reproduced in **1.2.6** and **1.3.0** (from git).

**Comment by cigamit** [comment]
Right, that will take care of itself once 1.2.x is merged into develop.  That's the way it works.  We are currently releasing from the 1.2.x branch.
